Examples/WindowsFormsApplication/ApplicationConfiguration.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Extensions.Configuration;
+using System;
+
+namespace WindowsFormsApplication
+{
+    public class ApplicationConfiguration
+    {
+        public string ClientId { get; private set; }
+        public string Tenant { get; private set; }
+        public string NfieldApiApplicationId { get; private set; }
+        public Uri NfieldApiUri { get; internal set; }
+        public string DomainName { get; internal set; }
+
+        public static ApplicationConfiguration Current { get; } = InitializeConfiguration();
+
+        private ApplicationConfiguration()
+        {
+        }
+
+        private static ApplicationConfiguration InitializeConfiguration()
+        {
+            var root = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+                .AddUserSecrets<Program>()
+                .Build();
+
+            return new ApplicationConfiguration
+            {
+                ClientId = root["Nfield:ClientId"],
```

Comment:
The "client" itself is something "external" to nfield I would say, so putting it in the configuration file under the Nfield namespace might be confusing. Maybe have a Nfield level where we put all nfield relevant config and a separate level where we put the "3rd party app" stuff. This might be over the top :-P.

Reply:
I'll put ClientId and TenantId in an `Application` section